functions/src/definitions/onVoteRequestUpdate.ts

```diff
@@ -56,26 +59,23 @@ exports.onVoteRequestUpdate = functions
       const voteTotal = await getCount(messageRef, "totalVoteScore")
       const truthScore = infoCount > 0 ? voteTotal / infoCount : null
       const thresholds = await getThresholds()
-      const isSus = susCount > parseInt(thresholds.isSus * responseCount)
+      const isSus = susCount > thresholds.isSus * responseCount
```

Comment:
any reason we not using parseInt? the answer will be different with the new one right

Reply:
The thresholds are numbers. typescript complains as `parseInt` expects string
```
const thresholds = {
  endVote: 0.5,
  endVoteSus: 0.2,
  endVoteUnsure: 0.8,
  startVote: 1,
  isSpam: 0.5,
  isLegitimate: 0.5,
  isInfo: 0.5,
  isIrrelevant: 0.5,
  isUnsure: 0.5,
  isSus: 0.5,
  falseUpperBound: 1.5,
  misleadingUpperBound: 3.5,
  sendInterimMinVotes: 1,
  surveyLikelihood: env === "DEV" ? 1 : 0.25,
  satisfactionSurveyCooldownDays: 30,
}
```